aexpect/client.py

```diff
@@ -586,9 +586,12 @@ def print_line(text):
                     break
                 if fd in r:
                     # Some data is available; read it
-                    new_data = os.read(fd, 1024).decode(self.encoding, "ignore")
+                    new_data = os.read(fd, 1024)
```

Comment:
Won't this bring back the bug that was fixed in https://github.com/avocado-framework/aexpect/commit/e935dfcfe06be713ff2908743e74b72b8e872d40 ?